src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSessionManager.cs

```diff
@@ -55,14 +56,33 @@ public async Task ActivateSessionAsync(int processId, bool runningUnderDebugger)
                 {
                     Process? process = Process.GetProcessById(processId);
 
-                    WriteOutputMessage(string.Format(VSResources.ProjectHotReloadSessionManager_AttachingToProcess, _pendingSessionState.Session.Name, processId));
+                    WriteOutputMessage(
+                        new HotReloadLogMessage(
+                            HotReloadVerbosity.Detailed,
+                            VSResources.ProjectHotReloadSessionManager_AttachingToProcess,
+                            projectName,
+                            _pendingSessionState.Session.Name,
+                            (uint)processId,
```

Comment:
Should the parameter be changed to `uint`?

Reply:
Yes, this method takes a uint, not an int. process id won't be negative so this is fine, right?

Reply:
I meant, should `processId` be declared as `uint` rather than its current `int`?